service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/PlacementAboutToSubmitEventControllerTest.java

```diff
@@ -42,15 +49,15 @@ void shouldUpdateChildPlacement() {
 
         DocumentReference child2NewApplication = testDocument();
         Element<Placement> child2NewPlacement = element(child2Placement.getId(),
-            testPlacement(child2, child2NewApplication));
+                testPlacement(child2, child2NewApplication));
 
         CaseDetails caseDetails = CaseDetails.builder()
-            .data(Map.of(
-                "children1", List.of(child1, child2),
-                "placements", List.of(child2Placement, child1Placement),
-                "placement", child2NewPlacement.getValue(),
-                "childrenList", child2.getId()))
-            .build();
+                .data(Map.of(
+                        "children1", List.of(child1, child2),
+                        "placements", List.of(child2Placement, child1Placement),
+                        "placement", child2NewPlacement.getValue(),
+                        "childrenList", child2.getId()))
+                .build();
```

Comment:
hm looks like your auto formatter is applying double indents when it shouldn't be, might want to reconfigure / double check